src/AppWrapper.style.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const AppWrapper = styled.div`
+  user-select: none;
+  max-width: 2200px;
+  display: flex;
+  & * {
+    box-sizing: border-box;
+  }
+`;
+
+export default AppWrapper;
```

Comment:
I think `App.style.js` would be a more appropriate name for this file?

Reply:
One can be confused that these styles are related to AppWrapper component or smth like that